client/my-sites/backup/backup-contents-page/file-browser/file-info-card.tsx

```diff
@@ -171,12 +171,12 @@ function FileInfoCard( {
 
 	const prepareDownloadClick = useCallback( () => {
 		if ( ! item.period || ! fileInfo?.manifestFilter || ! fileInfo?.dataType ) {
-			dispatch( onPreparingDownloadError() );
+			notices.showError( __( 'There was an error preparing your download. Please, try again.' ) );
```

Comment:
Similar to the comment above:

```suggestion
			notices.showError( __( 'There was an error preparing your download. Please try again.' ) );
```